lib/chat_api/conversations/helpers.ex

```diff
@@ -0,0 +1,54 @@
+defmodule ChatApi.Conversations.Helpers do
+  @moduledoc false
+
+  alias ChatApi.Slack
+
+  def send_conversation_state_update(conversation, state) do
+    conversation_state_message = get_conversation_state_message(state)
+
+    if conversation_state_message do
+      Slack.send_conversation_message_alert(
+        conversation.id,
+        conversation_state_message,
+        type: :convo_update
+      )
+
+      {:ok, conversation_state_message}
+    else
+      {:error, "state_invalid"}
+    end
+  end
+
+  def send_multiple_archived_updates(conversations) do
```

Comment:
is this used anywhere? if not, let's just remove it

Reply:
that's supposed to be in use. looks like I missed that 😬